src/AppShed/Extensions/StorageBundle/Entity/Repository/DataRepository.php

```diff
@@ -21,7 +21,7 @@ class DataRepository extends EntityRepository
      */
     public function getDataForView(View $view)
     {
-        return $this->getFilteredData($view->getStore(), $view->getFilters());
+        return $this->getFilteredData($view->getStore(), new ArrayCollection($view->getFilters()->toArray()));
```

Comment:
Why are you doing this?

you should be changing

    private function getFilteredData(Store $store, ArrayCollection $filters) {

to

    private function getFilteredData(Store $store, Collection $filters) {